readme.md

```diff
@@ -5,15 +5,15 @@
 _Si vous êtes à l'aise avec l'anglais n'hésitez pas à consulter [l'Awesome list
 Home Assistant](https://www.awesome-ha.com/) initiée et hebergé par @frenck._
 
-## Contents
+## Sommaire
```

Comment:
`Contents` section is mandatory to be compliant with awesome list requirement and being in the future referenced in the [master awesome list ](https://github.com/sindresorhus/awesome#readme)

I know it's not ideal for having a full list in French